tests/conftest.py

```diff
@@ -71,3 +72,11 @@ def autopatcher(request, target, **kwargs):
 @pytest.fixture
 def patch(request):
     return functools.partial(autopatcher, request)
+
+
+@pytest.fixture(scope="session", autouse=True)
+def factory_boy_random_seed():
+    # Set factory_boy's random seed so that it produces the same random values
+    # in each run of the tests.
+    # See: https://factoryboy.readthedocs.io/en/latest/index.html#reproducible-random-values
+    factory.random.reseed_random("hypothesis/lms")
```

Comment:
This is to make sure that if the tests pass when I run them, the tests will pass again if I run them again (without changing the code). Or if they pass for me, they'll pass for you and on CI too. We don't want flakey tests, so we don't want actual randomness in tests. The "predictable randomness" is enough to discourage tests from invisibly depending on fixed values having been passed by factories to models, while keeping the tests deterministic.